### PR TITLE
Changed store index page layout to grid etc.

### PR DESCRIPTION
**Pull request by simlar-0**
- Store index page now uses grid layout
- Fixed some pagination issues
- Restored cart and auth page links in navbar
- Generated some more products
- Minor visual changes to cart
- Updated database schema

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -69,24 +75,24 @@ CREATE TABLE IF NOT EXISTS `OrderStatus` (
...
-    `review_date` datetime  NULL ,
+    `review_date` datetime  NOT NULL ,
     `review_header` nvarchar(255)  NULL ,
     PRIMARY KEY (
         `customer_id`,`product_id`
     )
 );
 
-CREATE TABLE IF NOT EXISTS `CustomerPassword` (
+CREATE TABLE IF NOT EXISTS `CustomerPasswords` (
```
Comment (yuvve) [inline schema.sql]: All table names should either be singular or plural, but we shouldn't mix them in the same DB.

File: schema.sql [hunk 2]
```diff
@@ -69,24 +75,24 @@ CREATE TABLE IF NOT EXISTS `OrderStatus` (
...
     `customer_id` int  NOT NULL ,
     `hashed_password` binary(60)  NOT NULL ,
     PRIMARY KEY (
         `customer_id`
     )
 );
 
-CREATE TABLE IF NOT EXISTS `ManagerPassword` (
+CREATE TABLE IF NOT EXISTS `ManagerPasswords` (
```
Comment (yuvve) [inline schema.sql]: All table names should either be singular or plural, but we shouldn't mix them in the same DB.

Comment (simlar-0) [reply schema.sql]: fixed

File: schema.sql (hunk 1 above)
Comment (simlar-0) [reply schema.sql]: fixed

**Pull request merged**